src/modules/editor/ScriptEditorImplementation.cpp

```diff
@@ -216,6 +219,24 @@ void ScriptEditorWidget::asyncCompleterCreation()
 	}
 }
 
+void ScriptEditorWidget::highlightCurrentLine()
```

Comment:
+void ScriptEditorWidget::highlightCurrentLine() 

mix shoudl be then  +void ScriptEditorWidget::backgroundHighlight()  or no? why say highlightcurrent line in some and backgroundhighligh in others?

I know its code only and not visible to user...

 its exhausting trying to explain what I mean.